source/operatingsystems/windows/hmail/antispamsetup.md

```diff
@@ -0,0 +1,103 @@
+
+
+## Spam Tests
+
+![hMailServer > Settings > Anti-spam > Spam Tests](files/antispam/spamtests.png)
+
+- **Use SPF** - Default value is "3". This will only trigger when a domain hard-fails an SPF check. As such if you are seeing issues with spam, we would recommend changing this to 5. You can read further about SPF in our [main SPF guide](https://docs.ukfast.co.uk/operatingsystems/linux/mail/spf.html).
+
+- **Check host in the HELO Command** - Default value is "2". This will compare the hostname used in the sending mail server's HELO to its rDNS (PTR) record. We would recommend changing this to "3".
+
+- **Check that sender had DNS-MX records** - Default value is "2". This will make sure that the sender domain has MX reccordds. Any legitimate domain sending mail should have MX records. As such, we recommend setting this score to "5".
+
+- **Verify DKIM-Signature header** - Default value is "5". If the relevant sending domain has a DKIM DNS record, hMail will check to make sure the message's DKIM signature matches. If it does not, the check will fail. We recommend leaving this at the default value.
+
+```eval_rst
+.. note::
+
+  If no DKIM signature is included in the message's headers, this check will not be executed.
+
+```
+
+## SpamAssassin
+
+![hMailServer > Settings > Anti-spam > SpamAssassin](files/antispam/spamassassin.png)
+
+**SpamAssassin** is a popular, third-party mail filter that is updated regularly with additional rules for detecting spam messages. SpamAssassin requires some setup (allow approx. 20 minutes) but we would highly recommend it. An example implementation can be found on the hMailServer forum:
+
+https://www.hmailserver.com/forum/viewtopic.php?t=28133
+
+## DNS Blacklists
+
+![hMailServer > Settings > Anti-spam > DNS Blacklists](files/antispam/dnsblacklists.png)
+
+**DNS Blacklists** are lists maintained by different organisations that track IPs' associated mail traffic. When these are enabled it will check domains that email your server to ensure they are not on any blacklists. If the sending IP is blacklisted, it will reject the message. 
+
+hMail believe enabling this can cut down spam by as much as 15-25% and we would recommend enabling the default lists:
+
+- zen.spamhaus.org
+- bl.spamcop.net
+
+## SURBL Servers
+
+![hMailServer > Settings > Anti-spam > SURBL servers](files/antispam/surbl.png)
+
+**SURBL servers** work differently to DNS check lists by checking links in the body of the message. This will check incoming emails to see if they contain links to known-malicious websites. 
+
+## Grey Listing
+
+![hMailServer > Settings > Anti-spam > Greylisting](files/antispam/greylisting.png)
+
+**Grey Listing** works by _temporarily_ rejecting any mail sent to your server - a correctly configured mail server will almost always attempt to resend a message rejected in this manner, being successful on the second attempt. 
```

Comment:
Perhaps ' ... to your server. A correctly configured..."
or a semi-colon? '...to your server; a correctly configured..."

or just leave be.

Reply:
semi-colon it is